webpack/js/main.js

```diff
@@ -7,3 +7,4 @@ $(document).ready(function() {
         }
     });
 });
+//
```

Comment:
Could you add a newline at the end of file?

Reply:
Did it with the last commit. It's weird coz there is a newline in my local machine but when i am seeing it in github, there is no newline at the end. Can you please check?

Reply:
It's weird that in the last commit I have added a new line but when I view the commit, it doesn't show a newline and when i again try to edit it within github, new line appears.

Reply:
Can you please check?